make-release.sh

```diff
@@ -56,7 +59,10 @@ else
     # change back manifests to :latest
     find "${REPO_ROOT}/kubernetes-manifests" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
     git add "${REPO_ROOT}/kubernetes-manifests/*.yaml"
-    git commit -m "revert to latest images"
+    git commit -m "revert demo app images to latest"
+    find "${REPO_ROOT}/kubernetes-config" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
+    git add "${REPO_ROOT}/kubernetes-config/*.yaml"
+    git commit -m "revert loadgeenerator images to latest"
```

Comment:
This shouldn't have to be a new commit. You should be able to make the changes as part of the other one

Also, you need to make the `git add "${REPO_ROOT}/kubernetes-config/*.yaml"` part of the other commit too, to make sure it's in the release commit (put it under `# create release commit`)